.github/workflows/release.yml

```diff
@@ -20,12 +20,12 @@ jobs:
   build:
     runs-on: ubuntu-20.04
     env:
-      BUILDROOT_VERSION: '2021.02.4'
+      BUILDROOT_VERSION: '2021.08.2'
       BUILDROOT_DIR: 'buildroot'
 
     strategy:
       matrix:
-        raspberry: ['raspberrypi0', 'raspberrypi0w', 'raspberrypi4']
+        raspberry: ['raspberrypi0', 'raspberrypi0_2', 'raspberrypi4']
```

Comment:
It might make sense to rework the naming to keep them consistent. Buildroot uses `raspberrypizero2w`, so we might use `zero` too instead of `0`?

Or else `raspberrypi0`, `raspberrypi0v2`, `raspberrypi4` etc.